enterprise/frontend/src/embedding-sdk/components/public/InteractiveDashboard/EditableDashboard.unit.spec.tsx

```diff
@@ -340,4 +345,33 @@ describe("EditableDashboard", () => {
     expect(screen.getByText("Cancel")).toBeInTheDocument();
     expect(screen.getByText("Save")).toBeInTheDocument();
   });
+
+  it("should allow to create a new question", async () => {
+    await setup();
+    // These endpoints are used in the simple data picker
+    setupCollectionItemsEndpoint({
+      collection: createMockCollection(ROOT_COLLECTION),
+      collectionItems: [],
+    });
+    setupEmbeddingDataPickerDecisionEndpoints("flat");
+    setupSearchEndpoints([]);
+
+    expect(screen.getByTestId("dashboard-header")).toBeInTheDocument();
+
+    await userEvent.click(
+      within(screen.getByTestId("dashboard-header")).getByLabelText(
+        "Edit dashboard",
+      ),
+    );
+    await userEvent.click(
+      screen.getByRole("button", { name: "Add questions" }),
+    );
+    await userEvent.click(screen.getByRole("button", { name: "New Question" }));
+
+    // We should render the simple data picker at this point
+    expect(screen.queryByTestId("dashboard-header")).not.toBeInTheDocument();
+    expect(
+      await screen.findByRole("button", { name: "Pick your starting data" }),
```

Comment:
This is the end of the test as of now, more tests will be added with [later tasks](https://linear.app/metabase/project/allow-creating-a-new-question-right-from-editabledashboard-e8b02ef726ef/issues)